### PR TITLE
Bugfix: Hz -> GHz conversion for SetFrequency -> PulseQobjInstruction

### PR DESCRIPTION
**Pull request by SooluThomas**
### Summary

Fix the big pointed out in the [review comment](https://github.com/Qiskit/qiskit-terra/pull/4390#discussion_r420389769):

### Details and comments

`SetFrequency` should be in Hz on the frontend and has to be converted to GHz when `SetFrequency` is converted to Qobj Instruction and vice versa.

### CODE REVIEW

File: test/python/qobj/test_pulse_converter.py [hunk 1]
```diff
@@ -304,7 +304,7 @@ def test_set_frequency(self):
         """Test converted qobj from FrameChangeInstruction."""
         instruction = SetFrequency(8.0, DriveChannel(0))
 
-        qobj = PulseQobjInstruction(name='setf', ch='d0', t0=0, frequency=8.0)
+        qobj = PulseQobjInstruction(name='setf', ch='d0', t0=0, frequency=8e-9)
```
Comment (taalexander) [inline test/python/qobj/test_pulse_converter.py:307]: Would we fix the tests to that this is `qobj = PulseQobjInstruction(name='setf', ch='d0', t0=0, frequency=8.0)` and the pulse instruction is 8e9 please? This is more natural.

Comment (SooluThomas) [reply test/python/qobj/test_pulse_converter.py]: Fixed in https://github.com/Qiskit/qiskit-terra/pull/4399/commits/c70dd7c3c4626d46a45195b5e47ccbc2348ce479.

File: test/python/qobj/test_pulse_converter.py [hunk 2]
```diff
@@ -152,7 +152,7 @@ def test_set_frequency(self):
             name='setf',
             ch='d0',
             t0=0,
-            frequency=8.0
+            frequency=8e-9
```
Comment (taalexander) [inline test/python/qobj/test_pulse_converter.py:155]:
```suggestion
            frequency=8.
```
and line 149 should become `instruction = SetFrequency(8.0e9, DriveChannel(0))`

Comment (SooluThomas) [reply test/python/qobj/test_pulse_converter.py]: sorry, I missed it. Fixed in https://github.com/Qiskit/qiskit-terra/pull/4399/commits/33fe46d2fd645702fe5c412e01865aa3e852e46f.

**Pull request merged**